### PR TITLE
AB#86820 fixed fields mapping

### PR DESCRIPTION
**Pull request by rafalgajewski**

### CODE REVIEW

File: src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ private static SchemeEntity CreateEntity(AhpApplicationDto application, Uploaded
             SectionStatusMapper.ToDomain(application.schemeInformationSectionCompletionStatus),
             new AffordabilityEvidence(application.affordabilityEvidence),
             new SalesRisk(application.sharedOwnershipSalesRisk),
-            new HousingNeeds(application.meetingLocalProrities, application.meetingLocalHousingNeed),
+            new HousingNeeds(application.meetingLocalHousingNeed, application.meetingLocalProrities),
             new StakeholderDiscussions(
```
Comment (Antek012) [inline src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs:87]: This is ... just wrong

File: src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs [hunk 2]
```diff
@@ -83,7 +83,7 @@ private static SchemeEntity CreateEntity(AhpApplicationDto application, Uploaded
             SectionStatusMapper.ToDomain(application.schemeInformationSectionCompletionStatus),
             new AffordabilityEvidence(application.affordabilityEvidence),
             new SalesRisk(application.sharedOwnershipSalesRisk),
-            new HousingNeeds(application.meetingLocalProrities, application.meetingLocalHousingNeed),
+            new HousingNeeds(application.meetingLocalHousingNeed, application.meetingLocalProrities),
```
Comment (Antek012) [inline src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs:86]: You can rename fields in Domain and in View to have the same naming, will be simpler for future.

**Review by Carq — APPROVED** [review]

File: src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs (hunk 2 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs]: Fields are renamed

File: src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs (hunk 1 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.Domain/Scheme/Repositories/SchemeRepository.cs]: I reversed the change

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Models/Scheme/Factories/SchemeSummaryViewModelFactory.cs [hunk 3]
```diff
@@ -32,20 +32,20 @@ public SectionSummaryViewModel GetSchemeAndCreateSummary(string title, Contract.
                 scheme.HousesToDeliver?.ToString(CultureInfo.InvariantCulture).ToOneElementList(),
                 ActionUrl: CreateSchemeActionUrl(urlHelper, scheme.ApplicationId, nameof(SchemeController.Funding), allowWcagDuplicate: true)),
             new(
-                "Affordability od shared ownership",
+                "Affordability od Shared Ownership",
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Models/Scheme/Factories/SchemeSummaryViewModelFactory.cs:35]: od -> of

**Review by Antek012 — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Models/Scheme/Factories/SchemeSummaryViewModelFactory.cs (hunk 3 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.WWW/Models/Scheme/Factories/SchemeSummaryViewModelFactory.cs]: fixed

**Pull request merged**